typescript/scripts/README.adoc

```diff
@@ -0,0 +1,65 @@
+:toc: macro
+
+= tBTC v2 Refund script
+
+This script explains what to do in case you need to refund your BTC.
```

Comment:
Maybe a small note that `Deposit refunding is meant to be used only in the unlike scenario in which the tBTC could not be mined`?